docs/readme-cmake.md

```diff
@@ -1,88 +1,59 @@
 
-This will create a Visual Studio 2022 solution. If you are using an older version of VS, you will need to adjust the version. Now you will find a complete solution in the `build/` folder. Find the solution and open it and you're good to go!
+### Windows
 
-#### Using Microsoft Visual C++
+It is recommended to use the toolchain from Visual Studio to build the game (regardless of the IDE you wish to use).
 
-Using MSVC to compile Endless Sky is not supported. If you try, you will get a ton of warnings (and maybe even a couple of errors) during compilation.
+Download [Visual Studio](https://visualstudio.microsoft.com/downloads/#visual-studio-community-2022) (if you do not want to install Visual Studio, you can alternatively download the [VS Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2022)), and make sure to install the following components:
 
-<details>
+- "Desktop Development with C++",
+- "C++ Clang Compiler for Windows",
+- "C++ CMake tools for Windows".
 
-<summary>Building on Windows using MinGW</summary>
+It is recommended to use at least the version from 2022.
 
+### Windows (MinGW)
   
-You can download the [MinGW Winlibs](https://winlibs.com/#download-release) build, which also includes various tools you'll need to build the game as well. It is possible to use other MinGW builds as well.
-
-You'll need the MSVCRT runtime version, 64-bit. The latest version is currently gcc 12 ([direct download link](https://github.com/brechtsanders/winlibs_mingw/releases/download/12.1.0-14.0.4-10.0.0-msvcrt-r2/winlibs-x86_64-posix-seh-gcc-12.1.0-mingw-w64msvcrt-10.0.0-r2.zip)).
+You can download the [MinGW Winlibs](https://winlibs.com/#download-release) build, which also includes various tools you'll need to build the game as well. It is possible to use other MinGW distributions too (like Msys2 for example).
 
-Extract the zip file in a folder whose path doesn't contain a space (C:\ for example) and add the bin\ folder inside to your PATH (Press the Windows key and type "edit environment variables", then click on PATH and add it in the list).
+You'll need the POSIX version of MinGW. For the Winlibs distribution mentioned above, the latest version is currently gcc 13 ([direct download link](https://github.com/brechtsanders/winlibs_mingw/releases/download/13.2.0-16.0.6-11.0.0-ucrt-r1/winlibs-x86_64-posix-seh-gcc-13.2.0-mingw-w64ucrt-11.0.0-r1.zip)). Download and extract the zip file in a folder whose path doesn't contain a space (C:\ for example) and add the bin\ folder inside to your PATH (Press the Windows key and type "edit environment variables", then click on PATH and add it to the list).
 
-You will also need to install [CMake](https://cmake.org) (if you don't already have it).
-
-</details>
+You will also need to install a recent version of [CMake](https://cmake.org/download/). You can check if you have CMake already installed by running `cmake -v` in command line window.
```

Comment:
>  Is this included in "C++ CMake tools for Windows" or is it separate?

it's included yeah, should I mention this?

Reply:
>  Also, is this step required if using the Visual Studio installer to acquire the build tools?

Since this is in the MinGW section, I think we can assume that the user didn't install VS

Reply:
If it's included, then I don't think so. I recall we had some people who didn't seem to have CMake after installing Visual Studio and it was unclear if this package actually included all the CMake stuff needed.

Reply:
The package does include all the CMake stuff you need, idk what the issue was for those people.